templates/policies/privacy.hbs

```diff
@@ -0,0 +1,203 @@
+
+    <header>
+      <h2 id="docs-rs">docs.rs</h2>
+      <div class="highlight"></div>
+    </header>
+
+    <p>
+      <a href="https://docs.rs">Docs.rs</a> is managed by the members of the
+      <a href="{{baseurl}}/governance/teams/dev-tools#rustdoc">Rustdoc team</a>
+      and <a href="{{baseurl}}/governance/teams/core">Rust core team</a>. The
+      project collects and uses data as described in this privacy notice.
+    </p>
+
+    <p>
+      Docs.rs is an open source project to host documentation of crates for the
+      Rust Programming Language. It automatically builds crates' documentation
+      released on <a href="https://crates.io">crates.io</a> using the nightly
+      release of the Rust compiler. All information from crates that is
+      published on docs.rs is also available publicly on crates.io.
+    </p>
+
+    <header>
+      <h2 id="forums">Forums</h2>
+      <div class="highlight"></div>
+    </header>
+
+    <p>
+      <a href="{{baseurl}}/governance/teams/community">The Rust Community team</a>
+      administers the
+      <a href="https://users.rust-lang.org">Users Forum</a> and the
+      <a href="https://internals.rust-lang.org">Internals Forum</a>. Posts on
+      these forums are public. If you sign-up to participate in these forums, we
+      collect your email address and name. As administrators of the forum, we
+      have access to usage information regarding your interactions with it, such
+      as posts published and read, and time spent on the site.
+    </p>
+
+    <header>
+      <h2 id="third-party">Third-party Services</h2>
+      <div class="highlight"></div>
+    </header>
+
+    <p>
+      The Users Forum and Internals Forum on rust-lang.org are hosted by
+      <a href="https://www.discourse.org/">Discourse</a> and use its open source
+      discussion platform. Discourse’s privacy policy is available
+      <a href="https://www.discourse.org/privacy">here</a>.
+    </p>
+
+    <p>
+      We use Mailgun to send email. Mailgun’s privacy policy is available
+      <a href="https://www.mailgun.com/privacy-policy">here</a>.
+    </p>
+
+    <p>
+      GitHub login is used for authentication in Crates.io and (optionally) in
+      the forums. GitHub’s Privacy Statement can be found
+      <a href="https://help.github.com/en/articles/github-privacy-statement">here</a>.
+    </p>
+
+    <p>
+      Some Rust community members use the Zulip and Discord platforms for
+      community collaboration. Zulip’s privacy notice is available
+      <a href="https://zulipchat.com/privacy/">here</a>.
+      Discord’s privacy notice is available
+      <a href="https://discordapp.com/privacy">here</a>.
+    </p>
+
+    <header>
+      <h2 id="contact">Contact</h2>
+      <div class="highlight"></div>
+    </header>
+
+    <p>
+      For data subject access requests, or any questions about this privacy
+      notice, please email support at
+      <a href="mailto:privacy@rust-lang.org">privacy@rust-lang.org</a>
```

Comment:
Note that this email doesn't exist yet. Who should be on it? Leads of crates.io and docs.rs for sure, I assume we also want the core team on there. Anyone else?

Reply:
@skade Any thoughts on this?